metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -293,16 +293,16 @@ def get_inspectors(self) -> Iterable[Inspector]:
     def get_schema_names(self, inspector):
         return inspector.get_schema_names()
 
-    def get_workunits(self) -> Iterable[Union[MetadataWorkUnit, SqlWorkUnit]]:
+    def get_schema_names(self, inspector):
```

Comment:
this seems like a spurious edit

Reply:
Interesting, that's from a previous merge conflict that I had resolved locally. I've updated the file to reflect the current state. Passes local lintfix tests.